.github/config/ubuntu-22.04/conan/profiles/android-21-armv7

```diff
@@ -23,13 +29,32 @@ tools.android:ndk_path={{android_home}}/ndk/{{ndk_version}}
 AR={{android_home}}/ndk/{{ndk_version}}/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ar
 AS={{android_home}}/ndk/{{ndk_version}}/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-as
 RANLIB={{android_home}}/ndk/{{ndk_version}}/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-ranlib
-{% set cc = {
-    "armv7": "armv7a-linux-androideabi" + api_level + "-clang",
-    "armv8": "aarch64-linux-android" + api_level + "-clang",
-    "x86": "i686-linux-android" + api_level + "-clang",
-    "x86_64": "x86_64-linux-android" + api_level + "-clang",
-}[arch] %}
 CC={{android_home}}/ndk/{{ndk_version}}/toolchains/llvm/prebuilt/linux-x86_64/bin/{{cc}}
 CXX={{android_home}}/ndk/{{ndk_version}}/toolchains/llvm/prebuilt/linux-x86_64/bin/{{cc}}++
 LD={{android_home}}/ndk/{{ndk_version}}/toolchains/llvm/prebuilt/linux-x86_64/bin/ld
 STRIP={{android_home}}/ndk/{{ndk_version}}/toolchains/llvm/prebuilt/linux-x86_64/bin/llvm-strip
+
+[options]
+# gobject introspection not yet available for conan 2
+poppler/*:with_gobject_introspection=False
+
+# We are not using xorg functionality in cairo
+cairo/*:with_xlib=False
+cairo/*:with_xlib_xrender=False
+cairo/*:with_xcb=False
+cairo/*:with_symbol_lookup=False
+
+# Unused functionality in GLib
+glib/*:with_elf=False
+glib/*:with_mount=False
+glib/*:with_selinux=False
+
+# @TODO: These dependencies are currently problematic
+freetype/*:with_bzip2=False
+freetype/*:with_brotli=False
+poppler/*:with_tiff=False
+fontforge/*:with_tiff=False
+
+[replace_requires]
+libjpeg/9e: libjpeg/9f
+libdeflate/1.19: libdeflate/1.20
```

Comment:
is this specific to `os+os_version+arch`? if not we could try to push this into the packages or package clients

Reply:
I've removed `[replace_requires]`, it was a leftover due to version conflict, but seems to be solved now. All other config options moved to `pdf2htmlEX-config`, which is included from all the profiles. I don't want to put these options inside the packages because ideally we will not be having these packages in our repo - poppler, glib and et cetera, so there's no way to change the defaults. I can't put them in client options ( `self.requires("fontforge/20240423-git", options={"install_private_headers": True})` ) because a lot of those clients are once again - not our packages